docs/website/docs/hub/init.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Initialize a pipeline
+description: How to initialize and develop a dlt pipeline using CLI, workspace, or verified sources
+keywords: [create a pipeline, verified source, workspace, rest client, how to, dlt init]
+---
+
+# Initialize a pipeline
+
+This guide walks you through creating and initializing a `dlt` pipeline — whether manually via the CLI, through the LLM-native **dltHub workspace**, or from one of the community-maintained **verified sources**.
+
+## Overview
+
+A `dlt` pipeline moves data from a source (like an API or database) into a destination (like DuckDB, Snowflake, or Iceberg).
+You can create one in three ways:
+
+| Method | Command | Best for |
+|---------|----------|----------|
+| CLI-based | `dlt init <source> <destination>` | Developers who prefer manual setup |
```

Comment:
Should we call it CLI-based? All three require CLI usage. We can call it Manual or Template-based? I mean the only difference with the Verified source is that here you will have a template and you need to write the source yourself?